tei_config.rb

```diff
@@ -44,8 +44,8 @@
 
 ms_contents = 'tei:msContents'
 to_field 'cho_description', extract_tei("#{ms_desc}/#{ms_contents}/tei:summary")
-to_field 'cho_language', main_language
-to_field 'cho_language', other_languages
+to_field 'cho_language', main_language, transform(&:downcase), translation_map('not_found', 'languages', 'marc_languages')
+to_field 'cho_language', other_languages, transform(&:downcase), translation_map('not_found', 'languages', 'marc_languages')
```

Comment:
Can we remove https://github.com/sul-dlss/dlme-transform/blob/cf3dc0711aa1902233265da836b08b899c5e9d06/lib/macros/tei.rb#L32-L40 now?

Reply:
Nevermind, I'm missed that we're still using that.